Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG NODE_VERSION=18.17.0
-FROM --platform=linux/amd64 node:${NODE_VERSION}-alpine as build
+ARG IMAGE_NAME=18.19-alpine3.18
```

Comment:
if you just specify `node:18.19-alpine` you'll always get the latest version of `alpine` available for that node version